.travis.yml

```diff
@@ -1,25 +1,23 @@
-# Forces travis to use VM insted container, required to be able to build containers.
 sudo: required
-
 cache: pip
-
 services:
-    - docker
-
+- docker
 env:
-    matrix:
-        - JOB_TYPE=compile_and_integration_tests_fast
-        - JOB_TYPE=compile_and_integration_tests_slow
-
+  matrix:
+  - JOB_TYPE=compile_and_integration_tests_fast
+  - JOB_TYPE=compile_and_integration_tests_slow
+  - JOB_TYPE=compile_and_integration_tests_nightly
 before_install:
-    - sudo apt-get -qq update && sudo apt-get -qq install -y e2tools
-    - pip2.7 install -U --user paramiko fabric PyYAML pytest requests
-
+- sudo apt-get -qq update && sudo apt-get -qq install -y e2tools
+- pip2.7 install -U --user paramiko fabric PyYAML pytest requests
 script:
-    - if [ "$JOB_TYPE" = compile_and_integration_tests_fast ]; then
-        ( cd tests && bash run.sh --runfast );
-      fi
+- if [ "$JOB_TYPE" = compile_and_integration_tests_fast ]; then ( cd tests && bash
+  run.sh --runfast ); fi
+- if [ "$JOB_TYPE" = compile_and_integration_tests_slow ]; then ( cd tests && bash
+  run.sh --runslow ); fi
+- if [ "$JOB_TYPE" = compile_and_integration_tests_nightly ]; then ( cd tests && bash
+  run.sh --runnightly ); fi
 
-    - if [ "$JOB_TYPE" = compile_and_integration_tests_slow ]; then
-        ( cd tests && bash run.sh --runslow );
-      fi
+notifications:
+  slack:
+    secure: l831fv7p5iskdhG082XVdi5UBhYXEZyRn0ZgEfPWqLfqCj5FyxC36KaYiQo+vp8BS9vX0JFEOuZ4PiKjG3+jSEsJIk0Lx/1fjCg1zi13xfzI2ke1BWGuBT5tuz+nKj3TujfnHM0KE8Ix8GkwnamR388DO0FtgJrhEMBylo33zVV5ZESBa3B2bwUfWgfFF7VeUeqtDnOiHFDqsu2A4lOmMMJPp8k+71x0qvUsUStswrG11S0UhDReuuH1+siy3/bDEnrZIwEBvSM8DxdLLWC/SmCiCMdCU95GfzFRJsZN+2+KavCN8kXxvOG8rYrr8EhOLF4BbGDTJI9Dw+4H4OKOD/Bf+hr1OqYxpGtJxKzGWVlds1MwjjxklWlXopgIfwIZtmgC2RynLEw7LYHMcmBmr2+zjbvl0LZqWto1/G+la3J45rNaXawJtwY2RJ+c5jkm13BoYbOpUAg2SZ7WwW6frW6XdObiegpBfObSapAIEwMHeCSYvZbDPfhv0PFgGS9BQwJ+bz0PY1xoY45EaglnLt8zhcG3pfzXMo7jHFwIcP9/NGi7SO71lL4uhmcYn29PhRoIBMg/gjq7doycCfBWyUQrjGBOhDHzpq01TazxakDussWCSQ534ZwWrksI1d9ONMyikNOFXBkSHsqihYrNF+sLAtGEtHg0Ww7qRc0P/R0=
```

Comment:
Sorry, but this is still the wrong style. You're changing several lines that are already correct. The correct indentation (or at least the one we use) is four spaces per level.

Sorry to be pedantic about this, but stuff like this quickly adds up and makes our files less readable.